client/src/components/credentials/credentials.js

```diff
@@ -136,17 +136,17 @@ class Credentials extends React.Component {
   }
```

Comment:
Around line 87 looks like a left over `DESELECT_CREDENTIAL` in a dispatch. Fires when trying to delete multiple creds.